pyproject.toml

```diff
@@ -86,3 +86,5 @@ addopts = "--ignore=_build"
 env = [
     "BOOK_SERVER_CONFIG=test",
 ]
+# The _`default logging level` of INFO produces a LOT of output. Use a higher level of ``WARNING`` to reduce the noise.
+log_cli_level = 30
```

Comment:
is there no way to use `logging.WARNING`. It would be good to document what level logging is and how to override this from the cli.  It seems like a common workflow would be to run the tests, upon failure rerun the tests with the logging level set to debug to figure out what went wrong.

Reply:
I see that you did that below. sorry.

Reply:
I'd prefer to use `logging.WARNING`, bur got an error from pytest when I did. Sigh, that would be better.